editorial review (#1534)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc

Properties set: 10
----

Now import the movies.
This time, you will also create a relationship to the country in which the movie was made.
If you are storing your data in a SQL database, this is a one-to-many relationship type.

Use `MERGE` to create nodes that represent countries.

> This avoids creating duplicate country nodes in case multiple movies have been made in a same country.

Reviewer: ```suggestion
This avoids creating duplicate country nodes when multiple movies were made in the same country.
```
